src/main/resources/entity-types/invoice/composite_invoice__voucher_line__fund_distribution.json5

```diff
@@ -0,0 +1,59 @@
+{
+  id: '087bbe42-cc6d-40a9-8f41-c5fd6cee6037',
+  name: 'composite_invoice__voucher_line__fund_distribution',
+  private: false,
+  sources: [
+    {
+      id: 'b4c76df5-d730-4ede-8a98-ab04c5e222ab',
+      type: 'entity-type',
+      alias: 'voucher_line__fund_distribution',
+      useIdColumns: true,
+    },
```

Comment:
I know we talked about keeping the entity types as flat as possible, but I thought it might be good to have one composite entity type containing another composite one, just to show that it's possible.